.eslintcache

```diff
@@ -1 +1 @@
\ No newline at end of file
+[{"/Users/danielronan/Desktop/calculator-react/src/index.js":"1","/Users/danielronan/Desktop/calculator-react/src/reportWebVitals.js":"2","/Users/danielronan/Desktop/calculator-react/src/components/App.js":"3","/Users/danielronan/Desktop/calculator-react/src/components/ButtonPanel.js":"4","/Users/danielronan/Desktop/calculator-react/src/components/Display.js":"5","/Users/danielronan/Desktop/calculator-react/src/components/Button.js":"6"},{"size":300,"mtime":1612365564056,"results":"7","hashOfConfig":"8"},{"size":373,"mtime":1612363499000,"results":"9","hashOfConfig":"8"},{"size":180,"mtime":1612429388586,"results":"10","hashOfConfig":"8"},{"size":936,"mtime":1612429603155,"results":"11","hashOfConfig":"8"},{"size":228,"mtime":1612429370398,"results":"12","hashOfConfig":"8"},{"size":220,"mtime":1612429216777,"results":"13","hashOfConfig":"8"},{"filePath":"14","messages":"15","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"16"},"1puvdbs",{"filePath":"17","messages":"18","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"16"},{"filePath":"19","messages":"20","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"21","messages":"22","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"23","messages":"24","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"25","messages":"26","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/Users/danielronan/Desktop/calculator-react/src/index.js",[],["27","28","29","30","31","32","33"],"/Users/danielronan/Desktop/calculator-react/src/reportWebVitals.js",[],"/Users/danielronan/Desktop/calculator-react/src/components/App.js",[],"/Users/danielronan/Desktop/calculator-react/src/components/ButtonPanel.js",[],"/Users/danielronan/Desktop/calculator-react/src/components/Display.js",[],"/Users/danielronan/Desktop/calculator-react/src/components/Button.js",[],{"ruleId":"34","replacedBy":"35"},{"ruleId":"36","replacedBy":"37"},{"ruleId":"38","replacedBy":"39"},{"ruleId":"40","replacedBy":"41"},{"ruleId":"42","replacedBy":"43"},{"ruleId":"44","replacedBy":"45"},{"ruleId":"46","replacedBy":"47"},"jsx-a11y/accessible-emoji",[],"lines-around-directive",["48"],"no-spaced-func",["49"],"global-require",[],"no-buffer-constructor",[],"no-new-require",[],"no-path-concat",[],"padding-line-between-statements","func-call-spacing"]
```

Comment:
Let's add this to `gitignore` and delete the file from git. This way, we are not committing unnecessary files.